### PR TITLE
[v6r15] Polish RSS code with pylint

### PR DESCRIPTION
**Pull request by vipersec**
Please review my changes..

### CODE REVIEW

File: ResourceStatusSystem/Agent/CacheFeederAgent.py [hunk 1]
```diff
@@ -25,7 +25,7 @@ class CacheFeederAgent( AgentModule ):
   '''
 
   # Too many public methods
-  # pylint: disable-msg=R0904
+  # pylint: disable=R0904
```
Comment (chaen) [inline ResourceStatusSystem/Agent/CacheFeederAgent.py]: please use the short name rather than the code, it is easier to understand

**Comment by fstagni** [comment]
There is something wrong in your local settings, because the author of the commit appears to be... me!

**Comment by vipersec** [comment]
@fstagni Indeed, I will check again.

**Comment by atsareg** [comment]
Please, confirm that the changes are just pylint cosmetics, no real code changes. It is difficult to see the entire diff, Github refuses to show it.

**Comment by fstagni** [comment]
I confirm that. Review OK.

**Pull request merged**